src/Ads/AccountService.php

```diff
@@ -207,6 +212,51 @@ public function get_billing_status(): array {
 		];
 	}
 
+	/**
+	 * Gets the ads account access status.
+	 *
+	 * @return bool
+	 * @throws Exception When email is not present or invite link is empty when email present and has no account access.
+	 */
+	public function get_ads_account_has_access() {
```

Comment:
That makes sense @mikkamp . I have updated the function definition so that the moment we have the access to user's google ads' account, we will mark the `account_access` step as completed so that there would be no extra requests.